roast.it.js

```diff
@@ -1,3 +1,6 @@
-"use strict";
```

Comment:
Arcane way to code JavaScript, sir. I feel there is no need for magic when you can write clean code.

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Strict_mode